velox/core/PlanNode.h

```diff
@@ -324,6 +324,8 @@ class ValuesNode : public PlanNode {
   const size_t repeatTimes_;
 };
 
+using ValuesNodePtr = std::shared_ptr<const ValuesNode>;
```

Comment:
it would be nice to add the same alias to all other nodes, for consistency (in a separate PR).

Reply:
Sure, I can do that in a separate PR.